resources/views/layouts/partials/header/autocomplete.blade.php

```diff
@@ -8,61 +8,63 @@ class="{{ $inputClasses }}"
     v-on:focus="$root.loadAutocomplete = true; window.setTimeout(() => window.document.getElementById('autocomplete-input').focus(), 200)"
     v-if="!$root.loadAutocomplete">
 
-<autocomplete
-    v-if="$root.loadAutocomplete"
-    v-cloak
-    :additionals="{ categories: ['name^3', 'meta_description^1'] }"
->
-    <x-rapidez::reactive-base slot-scope="{ results, searchAdditionals, additionals }">
-        <data-search
-            placeholder="@lang('Search')"
-            v-on:value-selected="search"
-            component-id="autocomplete"
-            :inner-class="{ input: '{{ $inputClasses }}' }"
-            class="[&_*]:!m-0 [&_[isclearicon=]]:!mr-2"
-            :data-field="Object.keys(config.searchable)"
-            :field-weights="Object.values(config.searchable)"
-            :show-icon="false"
-            fuzziness="AUTO"
-            :debounce="100"
-            :size="10"
-            v-on:value-change="searchAdditionals($event)"
-        >
-            <div
-                slot="render"
-                slot-scope="{ downshiftProps: { isOpen }, data: suggestions }"
+<div class="w-full">
```

Comment:
Do we really need this one?

Reply:
Sadly yes, there is a div injected by ReactiveSearch that isn't properly sized otherwise.

Reply:
Correct and changed. ✅